sdk/src/main/java/io/radar/sdk/RadarJobScheduler.kt

```diff
@@ -74,10 +105,31 @@ class RadarJobScheduler : JobService() {
             this.jobFinished(params, false)
         }, 10000)
 
+        counter.set(0)
         return true
     }
 
     override fun onStopJob(params: JobParameters): Boolean {
+        if (Radar.isTestKey()) {
```

Comment:
Noticed that this logging is behind a test key check but the other ones aren't.  We might want to keep them consistent?

Reply:
Since we're using a bunch of new api's that we don't have alot of experience with (`PowerManager`, etc), I like the idea of putting these logs behind `Radar.isTestKey()`. That will let us dogfood these new apis without putting any prod users at risk of things like crashes, battery drain, slowness).  I doubt these api's are going to have any issues, but, always nice to roll things out safely in the SDK.

Reply:
Yeah I didn't want to add extra processing for regular users. the logging api is currently only used for non-test keys, so this made sense. I'll add some similar checks elsewhere, as suggested.

Reply:
done